### DISCUSSION TITLE
More detailed logging for cameras and webrtc

### DISCUSSION THREAD

**Discussion by dougransom** [blakeblackshear/frigate#8620] [Ask A Question]
**Describe what you are trying to accomplish and why in non technical terms**
I want to be able to see the URLs or other information on video streams, so I can restream video (perhaps to my neighbors who also have an NVR) or even  just know what frigate is doing.  So in the logs or UI panels (better) I would like links to the cameras, and the gortc video streams that are available, with hyperlinks i can even click on and test out in something like VLC.  

**Describe the solution you'd like**
Logging of every camera url and restreaming URL.   User interface  (Card) in HA that shows all of these as well.

**Describe alternatives you've considered**

**Additional context**
I have little knowledge of gortc but I could not access any restreamed url using VLC.

**Comment by NickM-27** [comment]
frigate itself can not do logging on go2rtc as that is all handled by go2rtc, you can of course increase the log level of go2rtc. The go2rtc webUI also already has a page that shows all of the different links for mse, webrtc, rtsp, http-flv, etc. etc,

Also, frigate 0.13 already supports setting a camera UI url which is then shown on the settings page

I am going to move this to a discussion as it seems like something is wrong with your setup given 

> I have little knowledge of gortc but I could not access any restreamed url using VLC.
